build/upgrade_annotations.sh

```diff
@@ -50,6 +50,10 @@ reset_to_master() {
         git remote add github-upstream git@github.com:box/box-content-preview.git || return 1
     fi
 
+    # Fetch latest code
+    git fetch github-upstream || return 1;
+    git checkout master || return 1
+
     # The master branch should not have any commits
     if [[ $(git log --oneline ...github-upstream/master) != "" ]] ; then
         echo "----------------------------------------------------"
```

Comment:
Change this error message to say something like you have extra commits in your master branch

Reply:
👍